notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<blockquote>Some parameters are fed to <code>predict_forecast</code> that ideally shouldn't need to be specified (like <code>seed</code> and <code>n_filters_factor</code>) and might seem contextually odd. They're used to locate the appropriate saved network. <em>This will be cleaned up in a future version</em>.</blockquote><p>Is the cleaned up version meant to be completed by this review? Or in a future as in after June 2024?</p><p> </p><p>Also, what does <code>n_filters_factor</code> do? Not immediately obvious. Edit: Seems to be documented in the 'Improving results' subsection below, maybe bring that up here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>This would be cleaned up in a future version after June 2024. Also, for <code>n_filters_factor</code>, added a brief description in parenthesis on first mention.</p>